src/main/scala/firrtl/passes/memlib/ReplaceMemTransform.scala

```diff
@@ -131,6 +134,20 @@ class ReplSeqMem extends SeqTransform with HasShellOptions with DependencyAPIMig
       helpText = "Blackbox and emit a configuration file for each sequential memory",
       shortOption = Some("frsq"),
       helpValueName = Some("-c:<circuit>:-i:<file>:-o:<file>")
+    ),
+    new ShellOption[String](
+      longOption = "gen-mem-verilog",
+      toAnnotationSeq = (a: String) =>
+        Seq(
+          a match {
+            case "blackbox" => GenVerilogMemBehaviorModelAnno(genBlackBox = true)
+            case _          => GenVerilogMemBehaviorModelAnno(genBlackBox = false)
+          },
+          RunFirrtlTransformAnnotation(new ReplSeqMem)
+        ),
+      helpText = "Blackbox and emit a Verilog file for each sequential memory",
```

Comment:
Blackbox and emit a Verilog behavior model for each sequential memory.

Reply:
Fixed in [`e0859e8` (#2202)](https://github.com/chipsalliance/firrtl/pull/2202/commits/e0859e84b3f507f47390b5b6838cb1b3fab45cdb)